### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.7 es 4.28 (30 ottobre)

### DISCUSSION THREAD

**Discussion by erinrrr** [sapienzastudentsnetwork/calcolo-delle-probabilita#308] [Esercizi M-Z]
![image](https://github.com/user-attachments/assets/39e7e9e2-9321-4167-ba58-c0f1b322d24a)

**Comment by erinrrr** [comment]
![image](https://github.com/user-attachments/assets/8e69d5fc-0302-4058-a06b-2eda7f67b7c7)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/5b995f64-5fef-4405-b35f-ba96cc730507)

**Comment by SickRickX2** [comment]
![image](https://github.com/user-attachments/assets/63583102-fece-4636-8adf-cede24b4e970)

**Comment by luckignolo32** [comment]
<img width="681" alt="{05604AC8-E9EC-46E6-A81B-F47D3EA30495}" src="https://github.com/user-attachments/assets/e30910f5-a9fb-4548-950b-edd0ca24e918" />
